src/components/ha-sidebar.html

```diff
@@ -102,19 +102,19 @@
     <paper-menu attr-for-selected='data-panel' selected='[[hass.currentPanel]]' on-iron-select='menuSelect'>
       <paper-icon-item on-tap='menuClicked' data-panel='states'>
         <iron-icon item-icon icon='mdi:apps'></iron-icon>
-        <span class='item-text'>States</span>
+        <span class='item-text'>[[tr(pg, "panel.states")]]</span>
       </paper-icon-item>
 
       <template is='dom-repeat' items='[[panels]]'>
         <paper-icon-item on-tap='menuClicked' data-panel$='[[item.url_path]]'>
           <iron-icon item-icon icon='[[item.icon]]'></iron-icon>
-          <span class='item-text'>[[item.title]]</span>
+          <span class='item-text'>[[tr(pg, item.title)]]</span>
```

Comment:
So this is an interesting case.

I think that we should not try to translate titles that are being send to us on the server. Wouldn't it be the responsibility of the component to register translations ? Otherwise custom components would require changes to the frontend to be translatable